rpc/wrtc_server_stream.go

```diff
@@ -263,6 +263,7 @@ func isContextCanceled(err error) bool {
 
 func (s *webrtcServerStream) processHeaders(headers *webrtcpb.RequestHeaders) {
 	s.logger = utils.AddFieldsToLogger(s.logger, "method", headers.Method)
+	s.logger.Debug("incoming grpc request")
```

Comment:
```suggestion
	s.logger.Debugw("Incoming gRCP request", "method", headers.Method)
```

Reply:
since we add the method as a field in the line above, the suggested change will log the method name twice in the same log line.

Reply:
Oops and I mispelled `grpc`. Ignore this comment.